topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+{: .question }
+
+> <details-title>More details on ideating learning outcomes and related information</details-title>
+> [Read about defining intended Learning Outcomes in the Design and plan session, course, materials tutorial available on GTN]( {% link topics/contributing/tutorials/design/tutorial.md#define-intended-learning-outcomes-los %} )
+{: .details}
+
+<a href="https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.1%20-%20Workflow%20stages%20description/411-prepare/" class="btn btn-dark text-white btn-lg btn-block">Go to the full description of FAIR-by-Design Methodology: Prepare stage ...</a>
+
+## Stage 2 - Discover
+
+> Greater even than the greatest discovery is to keep open the way to future discovery.
+{: .quote author="John Jacob Abel"}
+### Get inspired
+
+Reusable materials can be found anywhere. These are just some examples:
+
+> <code-in-title>GTN</code-in-title>
+> [GTN learning resources](https://training.galaxyproject.org)
+> 
+> [TeSS Catalogue by Elixir](https://tess.elixir-europe.org)
+{: .code-in}
+
+> <code-in-title>OER</code-in-title>
+> [DOAB](https://directory.doabooks.org/)
+> 
+> [MERLOT](https://www.merlot.org/merlot/index.htm)
+> 
+> [OASIS](https://oasis.geneseo.edu/index.php)
+> 
+> [OER Commons](https://www.oercommons.org/)
+> 
+> [OERTX CORA](https://www.projectcora.org/)
+> 
+> [GALILEO](https://oer.galileo.usg.edu/)
+> 
+> [FORRT](https://forrt.org/)
+{: .code-in}
+
+> <code-in-title>EOSC</code-in-title>
+> [EOSC Training catalogue on the EOSC Marketplace](https://search.marketplace.eosc-portal.eu/search/training?q=*)
+> 
+> Most EOSC projects have their own training catalogues and/or platforms.
+{: .code-in}
+
+> <code-in-title>General</code-in-title>
+> [Creative Commons Search](https://search.creativecommons.org/) - content provided under a CC license
+> 
+> [Zenodo](https://zenodo.org/) - a multi-disciplinary open repository
+> 
+> [OSF](https://osf.io/) - a free, open research platform
+{: .code-in}
+
+### Potential for reuse
+
+> <warning-title> Respect the licenses, to be respected!</warning-title>
+> - Materials with non-permissible licenses can be used for inspiration only.
+> - Materials with permissible licenses should be reused based on the license rules.
+{: .warning}
+
+### Don't forget the multimedia search
+
+Different learners have different learning modalities (read/write, auditory, visual, kinesthetic). To elevate the learning experience you should use all types of multimedia in your learning materials.
+
+<a href="https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.1%20-%20Workflow%20stages%20description/412-discover/" class="btn btn-dark text-white btn-lg btn-block">Go to the full description of FAIR-by-Design Methodology: Discover stage ...</a>
```

Comment:
```suggestion
[Go to the full description of FAIR-by-Design Methodology: Discover stage ...](https://fair-by-design-methodology.github.io/FAIR-by-Design_Book/4%20-%20FAIR-by-design%20learning%20materials%20creation/4.1%20-%20Workflow%20stages%20description/412-discover/){: .btn.btn-dark.text-white.btn-lg.btn-block}
```

i would recommend against adding all of those classes, not sure they will render nicely on dark mode, consider sticking to `btn` `btn-primary` and `btn-lg` if possible.